build.gradle

```diff
@@ -23,15 +23,15 @@
  */
 
 plugins {
-  id 'org.scm-manager.smp' version '0.10.1'
+  id 'org.scm-manager.smp' version '0.13.0'
 }
 
 dependencies {
   optionalPlugin "sonia.scm.plugins:scm-editor-plugin:2.0.0"
 }
 
 scmPlugin {
-  scmVersion = "2.0.0"
+  scmVersion = "2.19.0"
```

Comment:
This is required because i use the `Select` component instead the `Dropdown` and it was broken before 2.19.0...